diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/UnsafeUseLastIndexWarnTest.kt

```diff
@@ -0,0 +1,101 @@
+package org.cqfn.diktat.ruleset.chapter6
+
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.chapter6.UnsafeUseLastIndex
+import org.cqfn.diktat.util.LintTestBase
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class UnsafeUseLastIndexWarnTest : LintTestBase(::UnsafeUseLastIndex) {
+    private val ruleId = "$DIKTAT_RULE_SET_ID:last-index"
+
+    @Test
+    @Tag(WarningNames.UNSAFE_USE_LAST_INDEX)
+    fun `find method Length - 1 with many dot expressions`() {
+        lintMethod(
+            """
+                    |val A = "AAAAAAAA"
+                    |val D = A.B.C.length - 1
```

Comment:
could you please also add test for `.length - 2` and `.length`

Reply:
Done